src/components/Button/Button.tsx

```diff
@@ -3,16 +3,17 @@ import './Button.scss'
 
 type ButtonProps = {
     text?: string,
-    onClick: (event: React.MouseEvent<HTMLButtonElement>) => void,
+    type?: "button" | "submit" | "reset" | undefined
```

Comment:
undefined лишний за это отвечает символ "?"
```suggestion
    type?: "button" | "submit" | "reset" 
```